modyn/evaluator/internal/metrics/f1_score.py

```diff
@@ -70,4 +70,6 @@ def get_evaluation_result(self) -> float:
         return float(np.average(f1_scores, weights=total_labels_per_class / total_samples))
 
     def get_name(self) -> str:
+        if self.config.average == "binary":
+            return f"F1-{self.config.average}-{self.config.pos_label}"
```

Comment:
for f1 binary we can specify either 0 or 1 as the positive label. For those two metrics they should not have the same name, should be distinguished by pos label

Reply:
Can you add this as a comment please :)?

Reply:
will do